importer/Monument.py

```diff
@@ -106,6 +92,16 @@ def add_statement(self, prop_name, value, quals=None, refs=None):
         :param value: the value of the property
         :param quals: a dictionary of qualifiers
         :param refs: a list of references
+
+        Refs is None as default. This means that
```

Comment:
With a longer text like this I would have it above the parameters.

I would probably add `:param refs: a list of references or a single reference. Set None/True for the default reference, set False to not add a reference.`

Then describe the default above.